docs/reference/modules/scripting/using.asciidoc

```diff
@@ -236,6 +235,17 @@ DELETE _scripts/groovy/calculate-score
 // CONSOLE
 // TEST[continued]
 
+[float]
+[[modules-scripting-using-caching]]
+=== Script Caching
+
+All scripts are cached by default so that they do not need to be
```

Comment:
Suggested changes (mostly to remove passive voice): All scripts are cached by default so that they only need to be recompiled when updates occur. File scripts keep a static cache and will always reside in memory. Both inline and stored scripts are stored in a cache that can evict residing scripts. By default, scripts do not have a time-based expiration, but you can change this behavior by using the `script.cache.expire` setting. You can configure the size of this cache by using the `script.cache.max_size` setting. By default, the cache size is `100`.